# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/colord/hotfix-udev.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- colord-1.4.7/lib/colord/cd-edid.c.vanilla	2024-04-22 17:23:59.616000000 +0200
+++ colord-1.4.7/lib/colord/cd-edid.c	2024-04-22 17:23:02.890000000 +0200
@@ -118,7 +118,9 @@
 	udev = udev_new ();
 	if (udev == NULL)
 		goto out;
+#if 0
 	hwdb = udev_hwdb_new (udev);
+#endif
 	if (hwdb == NULL)
 		goto out;
 
